### DISCUSSION TITLE
what3words API requests

### DISCUSSION THREAD

**Discussion by fustygerbil** [strukturart/o.map#53] [Ideas]
Would you consider adding the ability to set location markers from what3words API requests?

API intro: https://what3words.com/en/news/industry/introducing-what3words-api-v3

It's disappointing that the implementation of this good idea is proprietary. But w3w addresses are used a lot by emergency services (that's how I'm exposed to them). So on the balance, it would be handy to have this feature.

What do you think?

**Comment by strukturart** [comment]
Thanks for your suggestion, I'll implement it soon, but I'm currently busy making this app: https://github.com/strukturart/flop available for every other browser to enable interactions between KaiOS devices and non-KaiOS devices. I plan to do the same thing with o.map.

**Reply by fustygerbil** [reply]
Cool, and flop is a really excellent idea. Thank you for making these solutions.

**Comment by strukturart** [comment]
I have integrated what3words in the marker menu. If you have created a marker with key 9 and jump between markers with key *, a request is made to what3words api, the answer then appears in the menu.

**Reply by fustygerbil** [reply]
Cool!
